docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -24,10 +32,41 @@ To create a new user:
 4. Review the new user information and note the temporary password that has been assigned. To send the user an introductory email, select the checkbox.
```

Comment:
@gal-revach I am not sure it is needed. The button changes based on the SSO flag. I can add it, it would be something like this: Press the *New User* or *New Local User* and enter... WDYT?

Reply:
@jasonnovichRunAI no, I agree, let's leave it as is.
However, we should remove the sentence "To send the user an introductory email, select the checkbox.", this wasn't developed